src/aiida/schedulers/plugins/bash.py

```diff
@@ -26,11 +26,12 @@ class BashCliScheduler(Scheduler, metaclass=abc.ABCMeta):
     def submit_job(self, working_directory: str, filename: str) -> str | ExitCode:
```

Comment:
Just something I noted: In many places the variable is called `working_directory`, while in others it's `workdir` now.

Reply:
Actually that's the only place that the variable is called `working_directory`, you'll find more `workdir` in the src. :smile: